language-plutus-core/src/Language/PlutusCore/Constant/Make.hs

```diff
@@ -100,56 +63,38 @@ Same considerations apply to bytestrings.
 -- | Convert a Haskell 'Integer' to the corresponding PLC @integer@.
 makeDynBuiltinInt
     :: TermLike term tyname name
-    => Type tyname () -- ^ An actual size @s@.
-    -> term ()        -- ^ A singleton for @s@.
-    -> Integer        -- ^ An 'Integer' to lift.
+    => Integer        -- ^ An 'Integer' to lift.
     -> term ()
```

Comment:
Let's remove this, it was only needed to make an integer that is sized just like some another integer.